src/react_native/wallet_connect.cljs

```diff
@@ -23,6 +23,8 @@
              :supportedNamespaces supported-namespaces})))
 
 (defn get-sdk-error
+  "Get an error from this list
+  https://github.com/WalletConnect/walletconnect-monorepo/blob/c6e9529418a0c81d4efcc6ac4e61f242a50b56c5/packages/utils/src/errors.ts"
```

Comment:
Maybe we could move this to a comment. Seems weird to have this url as a docstring